code/lib/core-common/src/utils/strip-abs-node-modules-path.ts

```diff
@@ -1,9 +1,14 @@
 import path from 'path';
-import { normalizePath } from 'vite';
+import slash from 'slash';
+
+function normalizePath(id: string) {
```

Comment:
This is copied largely from the `normalizePath` from `vite`, which was used previously in builder-vite.